main.go

```
@@ -0,0 +1,54 @@
+package main
+
+import (
+	"log"
+	"os"
+
+	"github.com/wmaldonadoc/academy-go-q42021/config"
+	"github.com/wmaldonadoc/academy-go-q42021/infrastructure/datastore"
+	"github.com/wmaldonadoc/academy-go-q42021/infrastructure/router"
+	"github.com/wmaldonadoc/academy-go-q42021/registry"
+
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+)
+
+func bootingGlobalLogger() *zap.Logger {
+	// Replacing logger global
+	config := zap.NewDevelopmentConfig()
+	config.EncoderConfig.EncodeLevel = zapcore.CapitalColorLevelEncoder
+	config.EncoderConfig.TimeKey = "timestamp"
+	config.EncoderConfig.EncodeTime = zapcore.ISO8601TimeEncoder
+	logger, _ := config.Build()
```

Never ignore errors.